components/provider_cldr/src/download/cldr_components.rs

```diff
@@ -1,6 +1,10 @@
 // This file is part of ICU4X. For terms of use, please see the file
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+
+//! Data downloader for CLDR JSON versions 37 and older. Prior to CLDR 38, the JSON for each
```

Comment:
I could have just deleted this code.  I don't like introducing legacy code so early in this project's lifetime.  Thoughts?

Reply:
my thought is that deleting the code would prevent comparison w/ CLDR 37, would be harder to track down issues.

Reply:
I'd remove the code. It's way to early for us to keep legacy code for any user who might want to rely on it. We should stay light on code and nimble as possible. Adding more legacy code to maintain in provider now is the opposite for me.

Reply:
@zbraniecki its' a small change, and easy enough for someone to pull out of the archives if need be.

No objection to deletion.

Reply:
How hard would it be to release old CLDR versions using the new zip file format, at least CLDR 36 and 37?

Reply:
I removed the CLDR 37 support now that ICU4X supports CLDR 38.